core/arch/arm/plat-k3/main.c

```diff
@@ -37,6 +37,10 @@ register_phys_mem_pgdir(MEM_AREA_IO_SEC, SEC_PROXY_SCFG_BASE,
 			SEC_PROXY_SCFG_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, SEC_PROXY_RT_BASE, SEC_PROXY_RT_SIZE);
 
+#ifdef DRAM0_BASE
+register_ddr(DRAM0_BASE, DRAM0_SIZE);
+#endif
```

Comment:
The `#ifdef` guards ain't required I think as the platform_config will always define these values based on the position of the `#define`